core/kazoo_convert/src/kz_fax_converter.erl

```diff
@@ -176,6 +186,10 @@ image_to_tiff(FromPath, #{<<"from_format">> := <<"image/tiff">>, <<"tmp_dir">> :
 image_to_tiff(FromPath, Options) ->
     convert_file(?CONVERT_IMAGE_COMMAND, FromPath, <<".tiff">>, Options).
 
+-spec image_to_pdf(kz_term:ne_binary(), map()) -> fax_converted().
```

Comment:
Is there a use case for this in faxing?

Reply:
For faxing is not.
I simple can see that all types of converters to `TIFF` and `PDF` is exist except of image_to_pdf.
This is reason why i added this.

Reply:
Since we don't have a use case for this, can you please revert this? I explicitly excluded cases that were not relevant for faxing in the kz_fax_converter as we plan on adding and moving more converters ro this lib in the future (various media conversions like wav to mp3, and other types of files for teletype notifications, port request documents etc..).  

I would prefer to keep the fax_converter cases specific to use cases related to faxing only. For faxing this type of conversion would always be from tiff to pdf and never other types of image to pdf since the PDF is always generated from the tiff we sent/received via the fax_smtp and fax_request.

Reply:
Ok I also will remove unneeded commits.